### DISCUSSION TITLE
108 課綱討論續

### DISCUSSION THREAD

**Discussion by quan0715** [1101-GS4719-C/Python-Community-GS4719-C#1510] [General]
（以下真實心境抒發，因為字太多了，我想要直接發一篇討論）
學習歷程根本就是一個強迫魚爬樹的制度，加劇貧富差距，讓社會走向唯利主義。 多少代表著青春活力的社團都倒在了108課綱，相對的真的這麼多人對資訊研究有興趣？對模擬聯合國有興趣？對生物研究有興趣？原本小小讀書會只招收“有興趣的人”，結果108課變相讓那些社團變成百人大社，幹部付出更多心血管理社團、規劃社課，又有多少人在認真聽講做專案，該改變的是這個大環境，為什麼不能靠藝術、靠電影、漫畫、運動天份來賺錢，而一定要讀醫牙電資，在資工系打滾了3年，看到好多好多，明明天賦就不是點在這上面（我也算有點到其他天賦），而被強留在這邊的資工咖，我們真的快樂嗎？自認在台灣人才得頂點，卻對自己的未來一片迷茫。
保握時間、珍惜光陰，每當我難過得時候，還可以抱著吉他，放聲高歌，但現在的小孩呢？只能靠解剖青蛙，或研究國際新聞取樂？總之，大家可以想想看到底如何改變台灣的大環境。

**Comment by ashleyy-huang** [comment]
我認為台灣的升學制度和讀書風氣都是很填鴨的，連社團活動、比賽都是為了履歷而參加，這導致很多學生需要花很多精力和時間去追求那些自己不喜歡、沒興趣的東西，也沒辦法專心在自己想要衝刺的地方，台灣的教育真的要好好的檢討一下，不可以再想要學習其他國家，學的四不像且還拖累了學生和家長

**Comment by youtinglin** [comment]
我也覺得學習歷程不適合台灣的社會，這使得很多家長或者學生變得很現實，只會考慮到所謂世俗的眼光，而忽略了自己真正喜歡做的事，甚至為了學習歷程無法冷靜思考自己的長才，就連參加個活動都要考慮對自己未來是否有利，反而沒辦法在該探索自我的年齡好好的了解心之所向。因為我本身在大學有籌辦給高中生的暑假營隊，發現有學生是因為對營隊領域有興趣而報名，但家長卻打來取消，說不希望小孩走這個領域，只因家長的不瞭解直接斷送小孩熱愛的事物。

**Comment by aa103967622** [comment]
我覺得這樣的政策真的只是為了做而做，沒有改變什麼根本的東西，要去做到適性發展，雖然你把學習歷程的佔分比例調高，那又怎樣？我們還是沒有辦法像國外一樣，有大把大把的時間好好發展我們有興趣的東西啊！終究還是徒勞......
